x-pack/plugin/build.gradle

```diff
@@ -170,6 +170,7 @@ tasks.named("yamlRestTestV7CompatTransform").configure { task ->
   task.skipTest("service_accounts/10_basic/Test get service accounts", "new service accounts are added")
   task.skipTest("spatial/70_script_doc_values/diagonal length", "precision changed in 8.4.0")
   task.skipTest("spatial/70_script_doc_values/geoshape value", "error message changed in 8.9.0")
+  task.skipTest("security/authz/14_cat_indices/Test empty request while single authorized index", "not supported for compatibility")
```

Comment:
Maybe replace `"not supported for compatibility"` with `"dataset.size added in 8.11.0"` or something like that? Could be a different PR, doesn't have to be this one. That is, if you're okay with that change but don't want to deal with another whole CI run on this PR, that's fine.